include/tvm/relay/dataflow_matcher.h

```diff
@@ -42,11 +42,16 @@ class DFPatternCallback;
 class DFPatternCallbackNode : public Object {
  public:
   /*! \brief Pattern this callback matches */
-  DFPattern pattern_;
+  DFPattern pattern;
   /*! \brief Function to call when finding a matched expression */
-  PackedFunc function_;
+  PackedFunc function;
+  /*! \brief Require InferType to be run before the callback */
+  bool require_type;
```

Comment:
https://tvm.apache.org/docs/contribute/code_guide.html
https://google.github.io/styleguide/cppguide.html#Variable_Names

Why the move away from the Google Style Guide convention? You seem to use the var_name_ convention in simplify_expr.cc.

Reply:
Because these variables are public, it's probably better and more consistent to name it without "_" at the end imo.